frontend/src/events.js

```diff
@@ -0,0 +1,5 @@
+// Enum of application events
```

Comment:
I don't know about having this file in the root source folder. Maybe we can create a folder to put this kind of thing in

Reply:
I guess the same could be said for a few other files in the root src folder

Reply:
Ok lets see discuss this at a time, but I probably wont address it in this PR